Branch-SDK/src/main/java/io/branch/referral/BranchContentUrlBuilder.java

```diff
@@ -31,6 +31,6 @@ public String getContentUrl() {
      * @param callback A {@link Branch.BranchLinkCreateListener} callback instance that will trigger
      */
     public void generateContentUrl(Branch.BranchLinkCreateListener callback) {
-        super.generateUrl(callback);
```

Comment:
part of https://github.com/BranchMetrics/android-branch-deep-linking-attribution/pull/888/files#r577363727